great_expectations/render/renderer/content_block/validation_results_table_content_block.py

```diff
@@ -357,14 +361,23 @@ def row_generator_fn(evr, styling=None, include_column_name=True):
                 unexpected_statement = cls._get_unexpected_statement(evr)
             except Exception as e:
                 logger.error("Exception occurred during data docs rendering: ", e, exc_info=True)
+                exception_traceback = traceback.format_exc()
+                exception_message = f'{type(e).__name__}: "{str(e)}".  Traceback: "{exception_traceback}".'
+                raise GreatExpectationsError(exception_message)
             try:
                 unexpected_table = cls._get_unexpected_table(evr)
             except Exception as e:
                 logger.error("Exception occurred during data docs rendering: ", e, exc_info=True)
+                exception_traceback = traceback.format_exc()
+                exception_message = f'{type(e).__name__}: "{str(e)}".  Traceback: "{exception_traceback}".'
+                raise GreatExpectationsError(exception_message)
```

Comment:
I have the same questions here about the re-raise.

Reply:
Same motivation as above.  Happy to discuss.